<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>911</width>
      <height>521</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QToolBox" name="toolBox">
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>130</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>128</width>
        <height>420</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Page 1</string>
      </attribute>
      <widget class="QSplitter" name="splitter">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>121</width>
         <height>111</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QToolButton" name="tBtnListIni">
        <property name="text">
         <string>初始化</string>
        </property>
       </widget>
       <widget class="QToolButton" name="tBtnListClear">
        <property name="text">
         <string>清除</string>
        </property>
       </widget>
       <widget class="QToolButton" name="tBtnListInsert">
        <property name="text">
         <string>插入</string>
        </property>
       </widget>
       <widget class="QToolButton" name="tBtnListAppend">
        <property name="text">
         <string>追加</string>
        </property>
       </widget>
       <widget class="QToolButton" name="tBtnDelete">
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>128</width>
        <height>420</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Page 2</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="page_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>128</width>
        <height>420</height>
       </rect>
      </property>
      <attribute name="label">
       <string>页</string>
      </attribute>
     </widget>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="labelCurrentChangedItem">
             <property name="text">
              <string>当前项的变化</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>208</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox">
             <property name="text">
              <string>可编辑</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QToolButton" name="tBtnSelectItem">
             <property name="minimumSize">
              <size>
               <width>46</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>项选则</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>删除</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="text">
              <string>添加</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget">
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>页</string>
      </attribute>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1032</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actInitList"/>
   <addaction name="actClear"/>
   <addaction name="actInsert"/>
   <addaction name="actAdd"/>
   <addaction name="actDelete"/>
   <addaction name="actExit"/>
   <addaction name="actItemSeclect"/>
   <addaction name="actAllSelect"/>
   <addaction name="actNoAllSelect"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQListWidget">
   <property name="text">
    <string>QListWidget操作</string>
   </property>
   <property name="toolTip">
    <string>QListWidget操作</string>
   </property>
  </action>
  <action name="actInitList">
   <property name="text">
    <string>初始化列表</string>
   </property>
  </action>
  <action name="actClear">
   <property name="text">
    <string>清除列表</string>
   </property>
  </action>
  <action name="actInsert">
   <property name="text">
    <string>插入</string>
   </property>
  </action>
  <action name="actAdd">
   <property name="text">
    <string>添加</string>
   </property>
  </action>
  <action name="actDelete">
   <property name="text">
    <string>删除当前项</string>
   </property>
  </action>
  <action name="actAllSelect">
   <property name="text">
    <string>全选</string>
   </property>
  </action>
  <action name="actNoAllSelect">
   <property name="text">
    <string>全不选</string>
   </property>
  </action>
  <action name="actInSeclet">
   <property name="text">
    <string>反选</string>
   </property>
  </action>
  <action name="actExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actItemSeclect">
   <property name="text">
    <string>项选择</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tBtnDelete</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>tBtnListIni</tabstop>
  <tabstop>tBtnListAppend</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>tBtnListClear</tabstop>
  <tabstop>tBtnListInsert</tabstop>
  <tabstop>pushButton_3</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>actItemSeclect</sender>
   <signal>triggered()</signal>
   <receiver>actInSeclet</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>515</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
